pkg/steps/release/promote.go

```diff
@@ -45,17 +44,14 @@ func (s *promotionStep) Inputs() (api.InputDefinition, error) {
 
 func (*promotionStep) Validate() error { return nil }
 
-var promotionRetry = wait.Backoff{
-	Steps:    20,
-	Duration: 10 * time.Millisecond,
-	Factor:   1.2,
-	Jitter:   0.1,
-}
-
 func (s *promotionStep) Run(ctx context.Context) error {
 	return results.ForReason("promoting_images").ForError(s.run(ctx))
 }
 
+const (
+	ciCentralRegistryDNS = "registry.svc.ci.openshift.org"
+)
+
 func (s *promotionStep) run(ctx context.Context) error {
```

Comment:
Should have some kind of test that spits out a sample yaml for this for easier inspection

Reply:
Updated.